packages/contracts/contracts/deploy/PayableDAppFactory.sol

```diff
@@ -50,6 +56,16 @@ contract PayableDAppFactory is
         transferOwnership(_owner);
     }
 
+    /// @notice sets a name for the pool using ENS service
+    function setName(string memory name) external override onlyOwner {
+        ReverseRegistrar ensReverseRegistrar = ReverseRegistrar(
```

Comment:
I think it's better to cast it as `IReverseRegistrar`, since we're not really concerned about the implementation, right?